charts/retool/templates/deployment_backend.yaml

```diff
@@ -54,7 +55,7 @@ spec:
 {{- end }}
 {{- end }}
       containers:
-      - name: {{ .Chart.Name }}
+      - name: main-backend
```

Comment:
container names are mutable, since the pods just get recreated as normal. using semantic container names helps log collection and metric collection, since deployment name isn't always available and pod name is dependent on nameOverride and such.

Reply:
let's go with "main-backend" even tho this has services bundled into it (including db connector)

"api" is not as discoverable in docs.